### PR TITLE
Fix default exports of JS globals

### LINKED ISSUE #5688: v2 build bug: "got unexpected undefined"
# 🐛 bug report

[Minimal repro repo here](https://github.com/shz/parcel-problem), which you can try for yourself using `npm i; npm run repro`.

When running `parcel build` with a certain configuration/import, the build fails with an error coming from the JS hoist module.

This is a _specific problem_ with the `@ungap/global-this` library using ES6 imports, and specifically _with the name_ `globalThis` as used in the library.  Editing the underlying source code to something similar like `globalWat` or `thatThis` resolves the issue. 

Running the equivalent code via `parcel serve` works just fine.

## 🎛 Configuration (.babelrc, package.json, cli command)

See the repro repo above, but here's the cole's notes:

#### package.json

```json
{
  "name": "parcel-problem",
  "version": "1.0.0",
  "scripts": {
    "repro": "parcel build --no-cache index.html"
  },
  "devDependencies": {
    "@ungap/global-this": "^0.4.3",
    "parcel": "2.0.0-nightly.540"
  }
}
```

#### index.html

```html
<!doctype>
<html>
  <body>
    <script src="./index.js"></script>
  </body>
</html>
```

#### index.js

```js
import globalThis from '@ungap/global-this';
```

## 🤔 Expected Behavior

The build should work, and all should be well.

## 😯 Current Behavior

The build fails with the following stack trace:

```
> parcel build --no-cache index.html

× Build failed.
@parcel/transformer-js: Got unexpected undefined
Error: Got unexpected undefined
    at nullthrows (C:\Users\lylep\Development\parcel-problem\node_modules\nullthrows\nullthrows.js:7:15)
    at safeRename (C:\Users\lylep\Development\parcel-problem\node_modules\@parcel\scope-hoisting\lib\hoist.js:1101:43)
    at MutableAsset.ExportDefaultDeclaration (C:\Users\lylep\Development\parcel-problem\node_modules\@parcel\scope-hoisting\lib\hoist.js:877:7)
    at NodePath._call (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\path\context.js:55:20)
    at NodePath.call (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\path\context.js:42:17)
    at NodePath.visit (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\path\context.js:92:31)
    at TraversalContext.visitQueue (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\context.js:115:16)
    at TraversalContext.visitMultiple (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\context.js:79:17)
    at TraversalContext.visit (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\context.js:141:19)
    at Function.traverse.node (C:\Users\lylep\Development\parcel-problem\node_modules\@babel\traverse\lib\index.js:82:17)
```

## 💁 Possible Solution

There's definitely something special about the `globalThis` name since renaming it (even to something similar) removes the problem.

## 🔦 Context

This is biting me in a more complex app on more recent versions of parcel v2, but the repro case (https://github.com/shz/parcel-problem) does a pretty good job of isolating it.  Hopefully it's helpful!

Possibly related to #5547 ?

## 💻 Code Sample

https://github.com/shz/parcel-problem

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-nightly > 415 |
| Node             | v14.4.0 |
| npm/Yarn         | npm 6.14.4 |
| Operating System | Windows 10 |

### PR DESCRIPTION
**Pull request by mischnic**
Closes #5688

Using `path.scope.hasBinding("...")` to check whether to rename a variable doesn't work because it also returns true for the JS language globals `globalThis`, `Set`...

Use `path.scope.hasBinding("...", true)` instead (`noGlobal = true`).

### CODE REVIEW

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.96s | -0.00ms |
| Cached | 595.00ms | -12.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 681.00ms | +55.00ms ⚠️ |
| dist/modern/index.f81b5b4b.js | 1.11kb | +0.00b | 690.00ms | +55.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 582.00ms | +38.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 686.00ms | +55.00ms ⚠️ |
| dist/modern/index.77ce15d4.css | 77.00b | +0.00b | 1.07s | +61.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/index.9ff7c523.js | 1.11kb | +0.00b | 58.00ms | -24.00ms 🚀 |
| dist/modern/index.f81b5b4b.js | 1.11kb | +0.00b | 81.00ms | +23.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 59.00ms | -25.00ms 🚀 |
| dist/legacy/index.5859ab37.css | 77.00b | +0.00b | 58.00ms | -23.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 20.66s | -47.00ms |
| Cached | 9.08s | +320.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.1964b8bb.js | 4.20kb | +0.00b | 1.02s | +59.00ms ⚠️ |
| dist/UserProfile.f5fa9f60.js | 1.76kb | +0.00b | 1.02s | +59.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 485.62kb | +0.00b | 100.00ms | +50.00ms ⚠️ |
| dist/PermalinkedComment.1964b8bb.js | 4.20kb | +0.00b | 100.00ms | +51.00ms ⚠️ |
| dist/UserProfile.f5fa9f60.js | 1.76kb | +0.00b | 100.00ms | +51.00ms ⚠️ |
| dist/NotFound.f4adce8d.js | 530.00b | +0.00b | 229.00ms | +187.00ms ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 87.00ms | +38.00ms ⚠️ |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.10m | -10.55s 🚀 |
| Cached | 3.14s | -53.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.25193e26.js | 2.46mb | -5.00b 🚀 | 1.16m | -10.38s 🚀 |
| dist/pdfRenderer.1c87caa3.js | 1.11mb | -6.00b 🚀 | 42.32s | -95.00ms |
| dist/editorView.94307fd1.js | 587.53kb | +0.00b | 57.92s | -22.12s 🚀 |
| dist/popup.f5b54d01.js | 169.25kb | +0.00b | 17.26s | +1.00s ⚠️ |
| dist/EmojiPickerComponent.b302072e.js | 139.01kb | +0.00b | 10.93s | -1.04s 🚀 |
| dist/card.92b0f4f9.js | 54.02kb | +0.00b | 11.88s | +3.07s ⚠️ |
| dist/Modal.507a0880.js | 39.18kb | +0.00b | 3.95s | -742.00ms 🚀 |
| dist/component.e862677c.js | 30.82kb | +0.00b | 3.81s | +1.64s ⚠️ |
| dist/esm.572878ca.js | 27.67kb | +0.00b | 7.49s | -4.39s 🚀 |
| dist/component.c1c3a8ff.js | 22.54kb | +0.00b | 3.94s | -748.00ms 🚀 |
| dist/DatePicker.64b5aa93.js | 21.20kb | +0.00b | 8.46s | +2.05s ⚠️ |
| dist/smartMediaEditor.ab51c6dd.js | 16.85kb | +0.00b | 17.25s | +2.86s ⚠️ |
| dist/js.63570256.js | 16.55kb | +0.00b | 3.94s | -745.00ms 🚀 |
| dist/dropzone.491f3481.js | 16.06kb | +0.00b | 12.28s | -3.94s 🚀 |
| dist/ui.bfd5ec70.js | 14.21kb | +0.00b | 11.21s | +2.72s ⚠️ |
| dist/workerHasher.c27b76fd.js | 11.90kb | +0.00b | 7.78s | -4.07s 🚀 |
| dist/component.dd6e8cc9.js | 6.20kb | +0.00b | 4.00s | +684.00ms ⚠️ |
| dist/EmojiPickerComponent.5d345edd.js | 3.62kb | +0.00b | 10.99s | +2.53s ⚠️ |
| dist/png-chunks-extract.8c8d635d.js | 3.55kb | +0.00b | 3.94s | -746.00ms 🚀 |
| dist/index.10f02a5e.css | 3.46kb | +0.00b | 42.32s | +23.77s ⚠️ |
| dist/dropzone.c3746b5e.js | 3.44kb | +0.00b | 11.52s | -4.69s 🚀 |
| dist/Modal.19c70720.js | 3.14kb | +0.00b | 1.12s | -1.89s 🚀 |
| dist/16.624bbe22.js | 2.49kb | +0.00b | 1.09s | -1.06s 🚀 |
| dist/ResourcedEmojiComponent.470ee09d.js | 2.15kb | +0.00b | 11.15s | +2.68s ⚠️ |
| dist/date.57372c70.js | 1.96kb | +0.00b | 4.49s | -485.00ms 🚀 |
| dist/images.c36248a4.js | 1.90kb | +0.00b | 4.63s | -1.02s 🚀 |
| dist/feedback.80c8a178.js | 1.86kb | +0.00b | 8.44s | +2.39s ⚠️ |
| dist/16.342e69d2.js | 1.86kb | +0.00b | 4.35s | -582.00ms 🚀 |
| dist/16.d173f3df.js | 1.79kb | +0.00b | 4.37s | +827.00ms ⚠️ |
| dist/workerHasher.f1855576.js | 1.75kb | +0.00b | 7.75s | -4.10s 🚀 |
| dist/workerHasher.cbff2c2b.js | 1.75kb | +0.00b | 16.97s | +2.70s ⚠️ |
| dist/list-number.9bb92c49.js | 1.68kb | +0.00b | 4.63s | -480.00ms 🚀 |
| dist/status.acecc9f0.js | 1.68kb | +0.00b | 4.92s | -415.00ms 🚀 |
| dist/code.6d1ec8ce.js | 1.61kb | +0.00b | 4.49s | -485.00ms 🚀 |
| dist/link.cda9435d.js | 1.53kb | +0.00b | 4.63s | -1.02s 🚀 |
| dist/heading6.f14573d2.js | 1.53kb | +0.00b | 8.44s | +2.39s ⚠️ |
| dist/heading3.dede5b4c.js | 1.51kb | +0.00b | 6.20s | +827.00ms ⚠️ |
| dist/16.adfefbb7.js | 1.46kb | +0.00b | 4.27s | +786.00ms ⚠️ |
| dist/emoji.5072e4af.js | 1.45kb | +0.00b | 4.50s | -483.00ms 🚀 |
| dist/16.186663a4.js | 1.45kb | +0.00b | 1.12s | -1.03s 🚀 |
| dist/16.45c1cffe.js | 1.40kb | +0.00b | 4.37s | -438.00ms 🚀 |
| dist/expand.739e3b88.js | 1.38kb | +0.00b | 8.44s | +2.37s ⚠️ |
| dist/16.8ff06a1d.js | 1.36kb | +0.00b | 4.36s | -581.00ms 🚀 |
| dist/16.263c691c.js | 1.34kb | +0.00b | 1.12s | -1.03s 🚀 |
| dist/heading2.5c4ee285.js | 1.33kb | +0.00b | 4.93s | -440.00ms 🚀 |
| dist/16.b7512655.js | 1.32kb | +0.00b | 4.35s | -1.14s 🚀 |
| dist/heading4.8bbff8c5.js | 1.29kb | +0.00b | 6.20s | +759.00ms ⚠️ |
| dist/Modal.d58fb436.js | 1.28kb | +0.00b | 3.94s | -746.00ms 🚀 |
| dist/16.436e2680.js | 1.27kb | +0.00b | 1.12s | -1.03s 🚀 |
| dist/layout.d9b11ad1.js | 1.27kb | +0.00b | 4.63s | -1.02s 🚀 |
| dist/16.eaa9cadb.js | 1.27kb | +0.00b | 4.55s | -386.00ms 🚀 |
| dist/16.e5432caf.js | 1.26kb | +0.00b | 4.17s | -464.00ms 🚀 |
| dist/16.62eaafc3.js | 1.26kb | +0.00b | 4.27s | +786.00ms ⚠️ |
| dist/quote.487996a9.js | 1.25kb | +0.00b | 4.92s | -333.00ms 🚀 |
| dist/component.3edf7abd.js | 1.23kb | +0.00b | 3.76s | +773.00ms ⚠️ |
| dist/action.8161ba48.js | 1.23kb | +0.00b | 4.49s | -319.00ms 🚀 |
| dist/16.0629cb86.js | 1.23kb | +0.00b | 3.76s | +773.00ms ⚠️ |
| dist/decision.cdcb6e3b.js | 1.21kb | +0.00b | 4.49s | -484.00ms 🚀 |
| dist/panel-warning.8b68f633.js | 1.21kb | +0.00b | 4.78s | -469.00ms 🚀 |
| dist/16.cd483156.js | 1.18kb | +0.00b | 4.35s | -450.00ms 🚀 |
| dist/list.e0e1a7a9.js | 1.18kb | +0.00b | 4.64s | -468.00ms 🚀 |
| dist/heading1.a939a2da.js | 1.18kb | +0.00b | 4.93s | -439.00ms 🚀 |
| dist/panel-error.210bc7a0.js | 1.11kb | +0.00b | 4.78s | -334.00ms 🚀 |
| dist/panel.8b5546b1.js | 1.10kb | +0.00b | 4.80s | -446.00ms 🚀 |
| dist/table.4e67906a.js | 1.09kb | +0.00b | 4.92s | -457.00ms 🚀 |
| dist/panel-success.a2e1eac9.js | 1.05kb | +0.00b | 4.78s | -469.00ms 🚀 |
| dist/panel-note.8e008c33.js | 1.05kb | +0.00b | 4.78s | -334.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.a20da0c5.js | 1003.00b | +0.00b | 12.23s | -3.98s 🚀 |
| dist/simpleHasher.d91ba9fe.js | 755.00b | +0.00b | 7.84s | -631.00ms 🚀 |
| dist/simpleHasher.b4d19bac.js | 755.00b | +0.00b | 17.24s | +2.96s ⚠️ |
| dist/index.html | 119.00b | +0.00b | 2.87s | +697.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.ec711acb.js | 2.43mb | -2.00b 🚀 | 590.00ms | -97.00ms 🚀 |
| dist/pdfRenderer.1c87caa3.js | 1.11mb | +0.00b | 657.00ms | -84.00ms 🚀 |
| dist/editorView.94307fd1.js | 587.53kb | +0.00b | 694.00ms | -61.00ms 🚀 |
| dist/popup.e96100d6.js | 169.17kb | +0.00b | 687.00ms | -48.00ms 🚀 |
| dist/EmojiPickerComponent.b302072e.js | 139.01kb | +0.00b | 656.00ms | -44.00ms 🚀 |
| dist/Toolbar.c11cfac5.js | 99.68kb | +0.00b | 694.00ms | -61.00ms 🚀 |
| dist/media-viewer.e8181fcd.js | 72.78kb | +0.00b | 657.00ms | -84.00ms 🚀 |
| dist/card.793062ce.js | 53.93kb | +0.00b | 650.00ms | -70.00ms 🚀 |
| dist/card.8b7fed58.js | 51.68kb | +0.00b | 689.00ms | -54.00ms 🚀 |
| dist/component.e862677c.js | 30.82kb | +0.00b | 589.00ms | -98.00ms 🚀 |
| dist/esm.572878ca.js | 27.67kb | +0.00b | 657.00ms | -43.00ms 🚀 |
| dist/DatePicker.2e53d84e.js | 21.12kb | +0.00b | 636.00ms | -67.00ms 🚀 |
| dist/smartMediaEditor.60d9f6fb.js | 16.76kb | +0.00b | 694.00ms | -61.00ms 🚀 |
| dist/dropzone.d74a9f7e.js | 16.00kb | +0.00b | 689.00ms | -54.00ms 🚀 |
| dist/ui.bfd5ec70.js | 14.21kb | +0.00b | 650.00ms | -70.00ms 🚀 |
| dist/workerHasher.c27b76fd.js | 11.90kb | +0.00b | 657.00ms | -43.00ms 🚀 |
| dist/card.89465567.js | 5.71kb | +0.00b | 657.00ms | -84.00ms 🚀 |
| dist/media-viewer.114c1cc8.js | 4.01kb | +0.00b | 678.00ms | -63.00ms 🚀 |
| dist/EmojiPickerComponent.5f66d6ae.js | 3.53kb | +0.00b | 648.00ms | -71.00ms 🚀 |
| dist/index.10f02a5e.css | 3.46kb | +0.00b | 693.00ms | -54.00ms 🚀 |
| dist/dropzone.c3746b5e.js | 3.44kb | +0.00b | 687.00ms | -48.00ms 🚀 |
| dist/Modal.19c70720.js | 3.14kb | +0.00b | 589.00ms | +44.00ms ⚠️ |
| dist/clipboard.84ebdbd2.js | 2.97kb | +0.00b | 689.00ms | -54.00ms 🚀 |
| dist/16.624bbe22.js | 2.49kb | +0.00b | 576.00ms | +31.00ms ⚠️ |
| dist/ResourcedEmojiComponent.470ee09d.js | 2.15kb | +0.00b | 650.00ms | -70.00ms 🚀 |
| dist/card.4188bb6a.js | 2.15kb | +0.00b | 687.00ms | -48.00ms 🚀 |
| dist/feedback.80c8a178.js | 1.86kb | +0.00b | 636.00ms | -67.00ms 🚀 |
| dist/browser.00e50e54.js | 1.82kb | +0.00b | 689.00ms | -54.00ms 🚀 |
| dist/workerHasher.f1855576.js | 1.75kb | +0.00b | 657.00ms | -43.00ms 🚀 |
| dist/workerHasher.cbff2c2b.js | 1.75kb | +0.00b | 694.00ms | -61.00ms 🚀 |
| dist/list-number.9bb92c49.js | 1.68kb | +0.00b | 611.00ms | -34.00ms 🚀 |
| dist/status.acecc9f0.js | 1.68kb | +0.00b | 613.00ms | -58.00ms 🚀 |
| dist/heading6.f14573d2.js | 1.53kb | +0.00b | 636.00ms | -66.00ms 🚀 |
| dist/heading3.dede5b4c.js | 1.51kb | +0.00b | 636.00ms | -50.00ms 🚀 |
| dist/16.186663a4.js | 1.45kb | +0.00b | 577.00ms | +32.00ms ⚠️ |
| dist/heading5.fe670803.js | 1.40kb | +0.00b | 636.00ms | -66.00ms 🚀 |
| dist/expand.739e3b88.js | 1.38kb | +0.00b | 636.00ms | -67.00ms 🚀 |
| dist/16.263c691c.js | 1.34kb | +0.00b | 577.00ms | +31.00ms ⚠️ |
| dist/heading2.5c4ee285.js | 1.33kb | +0.00b | 636.00ms | -50.00ms 🚀 |
| dist/mention.76dc783a.js | 1.29kb | +0.00b | 612.00ms | -33.00ms 🚀 |
| dist/heading4.8bbff8c5.js | 1.29kb | +0.00b | 636.00ms | -50.00ms 🚀 |
| dist/16.436e2680.js | 1.27kb | +0.00b | 577.00ms | +31.00ms ⚠️ |
| dist/quote.487996a9.js | 1.25kb | +0.00b | 613.00ms | -58.00ms 🚀 |
| dist/component.3edf7abd.js | 1.23kb | +0.00b | 589.00ms | -98.00ms 🚀 |
| dist/16.0629cb86.js | 1.23kb | +0.00b | 589.00ms | -99.00ms 🚀 |
| dist/media-card-analytics-error-boundary.3d09fdec.js | 1.23kb | +0.00b | 657.00ms | -84.00ms 🚀 |
| dist/decision.cdcb6e3b.js | 1.21kb | +0.00b | 602.00ms | -85.00ms 🚀 |
| dist/panel-warning.8b68f633.js | 1.21kb | +0.00b | 613.00ms | -52.00ms 🚀 |
| dist/list.e0e1a7a9.js | 1.18kb | +0.00b | 611.00ms | -34.00ms 🚀 |
| dist/heading1.a939a2da.js | 1.18kb | +0.00b | 636.00ms | -50.00ms 🚀 |
| dist/panel-error.210bc7a0.js | 1.11kb | +0.00b | 612.00ms | -34.00ms 🚀 |
| dist/panel.8b5546b1.js | 1.10kb | +0.00b | 613.00ms | -58.00ms 🚀 |
| dist/table.4e67906a.js | 1.09kb | +0.00b | 636.00ms | -50.00ms 🚀 |
| dist/panel-success.a2e1eac9.js | 1.05kb | +0.00b | 613.00ms | -52.00ms 🚀 |
| dist/panel-note.8e008c33.js | 1.05kb | +0.00b | 613.00ms | -33.00ms 🚀 |
| dist/media-picker-analytics-error-boundary.a20da0c5.js | 1003.00b | +0.00b | 687.00ms | -48.00ms 🚀 |
| dist/media-card-analytics-error-boundary.ef6306e0.js | 999.00b | +0.00b | 650.00ms | -71.00ms 🚀 |
| dist/media-card-analytics-error-boundary.889a764d.js | 999.00b | +0.00b | 689.00ms | -54.00ms 🚀 |
| dist/media-viewer-analytics-error-boundary.880deb8d.js | 995.00b | +0.00b | 687.00ms | -48.00ms 🚀 |
| dist/simpleHasher.d91ba9fe.js | 755.00b | +0.00b | 657.00ms | -43.00ms 🚀 |
| dist/simpleHasher.b4d19bac.js | 755.00b | +0.00b | 694.00ms | -61.00ms 🚀 |
| dist/index.html | 119.00b | +0.00b | 520.00ms | -101.00ms 🚀 |

</p></details>

<details><summary>Three.js ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 14.72s | +161.00ms |
| Cached | 663.00ms | +7.00ms |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/Three.js | 580.59kb | +0.00b | 117.00ms | +9.00ms ⚠️ |

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/288241956512858636)**

**Review by shz — APPROVED** [review]
[Not sure how normative this review should be, but I figured I'd chime in.]

This explanation for #5688 absolutely checks out, as does the fix (ignoring globals in the `hasBinding()` call).  The integration test and the fix for it look good to me!

File: packages/shared/scope-hoisting/src/hoist.js [hunk 1]
```diff
@@ -948,7 +948,10 @@ const VISITOR: Visitor<MutableAsset> = {
       }),
     );
 
-    if (isIdentifier(declaration) && path.scope.hasBinding(declaration.name)) {
+    if (
+      isIdentifier(declaration) &&
+      path.scope.hasBinding(declaration.name, true)
```
Comment (wbinnssmith) [inline packages/shared/scope-hoisting/src/hoist.js:953]:
```suggestion
      path.scope.hasBinding(declaration.name, /* noGlobal */ true)
```

**Review by wbinnssmith — APPROVED** [review]

File: packages/shared/scope-hoisting/src/hoist.js (hunk 1 above)
Comment (mischnic) [reply packages/shared/scope-hoisting/src/hoist.js:953]: Fixing Babel's API, one call at a time 😆

**Pull request merged**